wsproto/events.py

```diff
@@ -1,13 +1,20 @@
 # -*- coding: utf-8 -*-
 """
 wsproto/events
-~~~~~~~~~~
+~~~~~~~~~~~~~~
 
 Events that result from processing data on a WebSocket connection.
 """
 
 
-class ConnectionRequested(object):
+class Event(object):
+    """
+    Base class for wsproto events.
+    """
+    pass
```

Comment:
This new empty base class seems kind of pointless to me, but I guess it's harmless too... do you have a use case in mind?

Reply:
I probably had one back then...
hyper-h2 also uses one - seemed like a nice thing todo.